lms/djangoapps/grades/new/course_grade_factory.py

```diff
@@ -118,10 +126,17 @@ def iter(
 
     def _iter_grade_result(self, user, course_data, force_update):
         try:
+            kwargs = {
+                'user': user,
+                'course': course_data.course,
+                'collected_block_structure': course_data.collected_structure,
+                'course_key': course_data.course_key
+            }
+            if force_update:
+                kwargs['force_update_subsections'] = True
+
             method = CourseGradeFactory().update if force_update else CourseGradeFactory().create
-            course_grade = method(
-                user, course_data.course, course_data.collected_structure, course_key=course_data.course_key,
-            )
+            course_grade = method(**kwargs)
```

Comment:
You can revert these changes per my comment above regarding not needing to pass `force_update_subsections` to the CourseGradeFactory's update method - since it's assumed from the method's contract.